event_sourcery_django/__init__.py

```diff
@@ -22,8 +24,15 @@
 from event_sourcery.event_store.outbox import Outbox
 
 
+class Config(BaseModel):
+    model_config = ConfigDict(extra="forbid", frozen=True)
```

Comment:
What about creating something like global `ImmutableBaseModel` class?

Reply:
I'm not sure. Creating a model, that in the long term can grow with some addons, or having a config. Imo config is short and explicit.